kernel/Kconfig

```diff
@@ -279,6 +279,13 @@ config WAITQ_DUMB
 
 endchoice # WAITQ_ALGORITHM
 
+config CODE_DATA_RELOCATION
```

Comment:
why is this under kernel/Kconfig? Please move out of here to Kconfig.zephyr under linker options for example.